src/scripts/run-build.sh

```diff
@@ -23,7 +23,14 @@ if [[ $os = CYGWIN* || $os = MINGW* || $os = MSYS* ]]; then
 fi
 
 # create buildtool command from parameters
-buildCommand="buildtool ${PARAM_TASKS}"
+buildCommand="buildtool"
+
+if [ -n "$PARAM_TASKS" ]; then
+    buildCommand+=" ${PARAM_TASKS}"
+fi
+if [ -n "$PARAM_BUILD_OPTIONS" ]; then
+    buildCommand+=" ${PARAM_BUILD_OPTIONS}"
```

Comment:
Do we want the flags to come before the tasks or after? I normally pass in flags before the list of tasks.

Reply:
We document putting them after: https://www.mathworks.com/help/matlab/ref/buildtool.html

Also for compatibility/ clarity `-ignoreUnknownOptions` generally needs them at the end if we plan to use that at any point.